### PR TITLE
Rename the library to Judoscale

### PR DESCRIPTION
**Pull request by carlosantoniodasilva**

### CODE REVIEW

File: judoscale.gemspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'judoscale/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "judoscale"
```
Comment (carlosantoniodasilva) [inline judoscale.gemspec:7]: Name is subject to change.

File: judoscale.gemspec [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.require_paths = ["lib"]
+
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.metadata = {
+    "homepage_uri" => "https://judoscale.com",
+    "bug_tracker_uri" => "https://github.com/judoscale/judoscale-ruby/issues",
+    "documentation_uri" => "https://judoscale.com/docs",
```
Comment (carlosantoniodasilva) [inline judoscale.gemspec:24]: Some of these /docs URLs are subject to change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,114 +1,114 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/adamlogic/rails_autoscale_agent/compare/v0.10.2...master)
+## [Unreleased](https://github.com/judoscale/judoscale-ruby/compare/v0.10.2...master)
 
 No currently unreleased changes.
 
-## [0.10.2](https://github.com/adamlogic/rails_autoscale_agent/compare/v0.10.1...v0.10.2) - 2021-01-12
+## [0.10.2](https://github.com/judoscale/judoscale-ruby/compare/v0.10.1...v0.10.2) - 2021-01-12
```
Comment (carlosantoniodasilva) [inline CHANGELOG.md:8]: All the same commit/tag references apply to the fork, so we can just point everything to it.

An alternative would be to keep the refs, assuming the original repo is not going anywhere, and add a changelog entry for the change. (which I think is a good idea anyway.)

Comment (adamlogic) [reply CHANGELOG.md:8]: I think I'm okay to change these refs. Good idea to add a changelog entry.

**Review by adamlogic — APPROVED** [review]
Looks great!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,114 +1,114 @@
...
 
 ### Changed
 
-- Loosen Ruby constraint to allow Ruby 3. ([#36](https://github.com/adamlogic/rails_autoscale_agent/pull/36))
+- Loosen Ruby constraint to allow Ruby 3. ([#36](https://github.com/judoscale/judoscale-ruby/pull/36))
```
Comment (carlosantoniodasilva) [inline CHANGELOG.md:12]: @adamlogic I just realized that PRs reset so this (and a handful others) needs to point to the original repo, to a SHA, or the reference removed. Any preferences on that?

Comment (carlosantoniodasilva) [reply CHANGELOG.md]: I reverted the changes on the changelog so we can keep the references there pointing to the original PRs/issues/SHAs. I think we can circle back on it if we want to point to specific SHAs across the board in the changelog then, but it's not like the original repo is going anywhere any time soon.

File: CHANGELOG.md (hunk 3 above)
Comment (carlosantoniodasilva) [reply CHANGELOG.md]: changelog entry added. 

https://github.com/judoscale/judoscale-ruby/pull/1/commits/7bbcfdaaad0b89c7fa139871be09324396d6b03b

**Pull request merged**